packages/core/src/app.ts

```diff
@@ -1524,11 +1536,36 @@ export class App {
         }
     ) {
         const { vault } = this.getItem(item.id)!;
-        vault.items.update(new VaultItem({ ...item, ...upd, updatedBy: this.account!.id }));
+        const newItem = new VaultItem({ ...item, ...upd, updatedBy: this.account!.id });
+        this._updateHistory(item, newItem);
+        vault.items.update(newItem);
```

Comment:
I assume/believe something here is removing the `.name` and `.tags` properties from `item.history[]` entries. I suppose it's serialization, but haven't found where @MaKleSoft .